monster-cards/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+	"version": "2.0.2",
+	"compilerOptions": {
+		"declaration": false,
+		"module": "umd",
+		"noImplicitAny": true,
+		"outDir": "_build/",
+		"baseUrl": "node_modules",
+		"removeComments": false,
+		"sourceMap": true,
+		"target": "es5",
+		"moduleResolution": "classic"
```

Comment:
lets change this to `node`, it shouldn't in theory affect anything at this point